templates/Dockerfile.erb

```diff
@@ -47,6 +49,10 @@ RUN rm /usr/lib/ruby/gems/2.3.0/gems/facter-"$FACTER_VERSION"/lib/facter/blockde
 <% end %>
 <% end %>
 
+<% if use_factfile %>
+    COPY <%= factfile %> /etc/puppetlabs/facter/facts.d/custom_facts.txt
```

Comment:
Wanted all the facts to be at a location that are passed as a parameter. I can change it and ask user to enter the path for creation/checking facts.d folder and adding the facts file into it